validation/standard/init.go

```diff
@@ -67,7 +74,7 @@ func decodeTOMLFileIntoConfig[T Params | Roles | MultisigRoles | VersionTags | B
 // LoadImmutableReferences parses standard-immutables.toml and stores it in a map. Needs to be invoked one-time only.
 func LoadImmutableReferences() {
 	var bytecodeImmutables *ContractBytecodeImmutables
-	for tag := range Versions.Releases {
+	for tag := range NetworkVersions["mainnet"].Releases {
```

Comment:
@geoknee How should I deal with `LoadImmutableReferences` here?

Reply:
Let's get some input from @vdamle on this. I think what we are doing here is  crafting an object with only the immutable references for some of the tags. Might be less indirect to just use the mapping keyed by the version tag as we do for the versions, and slice into it with the `StandardRelease`.

Reply:
@vdamle AFAIK, immutables _do not_ vary between sepolia and mainnet. 

Some additional context on this PR. We need to get this PR merged as part of our OPCM/op-deployer deadline on Friday. Let me know what I can do to push this over the line. 🙏🏻

Reply:
Since we are only interested in validating bytecode for the `StandardVersion` (let me know if that's incorrect @geoknee ), I think the following should suffice:

```
	for contractVersion, immutables := range BytecodeImmutables {
		if contractVersion == NetworkVersions["mainnet"].StandardRelease { // we want to check the bytecode for standard release version
			bytecodeImmutables = &immutables
			break
		}
	}
```

Reply:
Agreed that the immutable references will not vary between sepolia and mainnet. Also, since we ignore the actual value of the immutables in our checks, this means that `standard-bytecodes.toml` also does not need to be split between sepolia and mainnet. 

I did some tidy up work here https://github.com/ethereum-optimism/superchain-registry/pull/616 which we could merge into this PR, or as a fast follow into `main`.

Reply:
Resolving this thread, thanks folks.